.gitlab-ci.yml

```diff
@@ -384,7 +384,7 @@ test_acceptance_ubuntu_raspberrypi3:
 .template:publish:docker-image:
   stage: publish
   tags:
-    - docker
+    - hetzner-amd-beefy
```

Comment:
I did not research further, but changing yesterday jobs from public `docker` tags to `hetzner-amd-beefy` did not work for `mender-artifact` (link [here](https://github.com/mendersoftware/mender-artifact/pull/610#issuecomment-2097861179)) due to the runners having different Docker in Docker setup.

This job also we cannot test until we merge the PR, so I am okey with merging as-is as long as you keep an eye on the master pipeline :+1:

Reply:
It looks like the `gitlab-org-docker` runner is [actually using privileged runners](https://docs.gitlab.com/ee/development/pipelines/internals.html#runner-tags). I think this is bad for a security point of view. Let me check in master, then

Reply:
The issues could be caused by the custom MTU setting: see https://github.com/mendersoftware/mender-convert/pull/665